definitions/pipelines/germline_exome.cwl

```diff
@@ -134,9 +134,6 @@ outputs:
     verify_bam_id_depth:
         type: File
         outputSource: alignment_and_qc/verify_bam_id_depth
-    gvcf:
-        type: File[]
-        outputSource: detect_variants/gvcf
```

Comment:
If the gvcf stuff is being removed, are inputs like `gvcf_gq_bands: gvcf_gq_bands` still being used?

Reply:
They would still be used for other pipelines using the gatk_haplotypecaller.cwl in 'GVCF' mode such as whole genome sequencing. For the single sample germline pipeline, the input for gvcfs_gq_bands is still there, but I don't believe it is actually influencing the calls as haplotypecaller is not being run in GVCF mode. I left the input so that other pipelines can still use the same haplotypecaller.cwl file.